pipeline/git_preflight.go

```
@@ -313,6 +389,28 @@ func runAutoInit(ctx context.Context, workDir string, allowInit bool, interactiv
+	// stage their own initial commit. Empty workdir → fall through to
+	// the empty-commit path below.
+	hasContent, contentErr := workdirHasContent(workDir)
+	if contentErr != nil {
+		return fmt.Errorf("%w: scan workdir: %v", ErrGitAutoInitRefused, contentErr)
+	}
+	if hasContent {
+		// Use %q for workDir so paths containing spaces / special chars
+		// produce copy/pasteable commands (Copilot:3260796955,
+		// CodeRabbit:3260803559). %s here would emit unquoted shell-broken
+		// commands for any non-trivial workdir path on macOS/Windows.
+		return fmt.Errorf("%w: workdir is not empty.\n\n  --git=init creates an empty initial commit; tracker will not auto-`git add` your existing files (they might be secrets, build artifacts, or content you haven't decided to track). Stage and commit them yourself so you control the baseline:\n\n    git -C %q init\n    git -C %q add .\n    git -C %q commit -m \"initial\"\n\n  Or empty the workdir first, then re-run with --git=init --allow-init.", ErrGitAutoInitRefused, workDir, workDir, workDir)
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not end with punctuation or newlines

(staticcheck)